Extensions/xinbox.js

```diff
@@ -1064,6 +1088,47 @@ XKit.extensions.xinbox = new Object({
 		document.body.appendChild(script);
 	},
 
+	do_post_colors: function(){
+		var colors = XKit.extensions.xinbox.get_blog_colors();
+		$(".posts .post[data-tumblelog]").not(":has(.xinbox-color-strip)").each(function(j,i){
```

Comment:
We don't seem to be using `j` here. Would it make sense to name this parameter `_` to denote that it's unused instead?